04LessonFour/in-class-project/index.html

```diff
@@ -53,7 +53,7 @@
          <div class="img2">
 
                <h1>Architect of the Future</h1>
-               <p style="padding-left: 5px">Intel strives to outdo current technology by supporting end-to-end solutions and designing system architecture that meets impending neends while exceeding currrent ones. See how innovative technologies such as Intel Solid-State Drives are leading to a future of endless possibilites.</p>
+               <p>Intel strives to outdo current technology by supporting end-to-end solutions and designing system architecture that meets impending neends while exceeding currrent ones. See how innovative technologies such as Intel Solid-State Drives are leading to a future of endless possibilites.</p>
          </div>
 

```

Comment:
NO INLINE STYLING!!   Also, if you need the image to come first on the page, it needs to come first in the html. make sure you're setting your image width to be 100% so it doesn't overlap anything.